### PR TITLE
mount overlayfs into diskimage to avoid problems with ftype=0

### PR DESCRIPTION
**Pull request by jmikovic**

### CODE REVIEW

File: repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/libraries/preparetransaction.py [hunk 1]
```diff
@@ -163,3 +163,34 @@ def mount_dnf_cache(overlayfs_info):
...
 def umount_dnf_cache(overlayfs_info):
     return check_cmd_call(['/bin/umount', os.path.join(overlayfs_info.merged, 'var', 'cache', 'dnf')])
+
+
+def create_disk_image(path):
+    diskimage_path = os.path.join(path, 'diskimage')
+    mounts_path = os.path.join(path, 'mounts')
+
+    if os.path.isdir(mounts_path):
+        check_cmd_call(['/bin/umount', '-fl', mounts_path])
```
Comment (Rezney) [inline repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/libraries/preparetransaction.py]: You could give "remove_disk_image" function an "ignore_errors" parameter and then call it here.

Comment (Rezney) [inline repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/libraries/preparetransaction.py]: I know it is not related to this PR but do not we have already some stlib stuff for calling external programs?

Comment (vinzenz) [reply repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/libraries/preparetransaction.py]: the function check_cmd_call should have been refactored in #13 to use stdlib.call but it wasn't so yeah this needs to be done in a different PR

**Comment by vinzenz** [comment]
@Rezney If we always go for it, we don't need to check for all the cases where this will be necessary. So I guess yeah always is probably better just to ensure it will work all the time and we'd find issues with this approach too.

**Comment by vinzenz** [comment]
@jmikovic Please rebase and solve the conflicts

**Comment by jmikovic** [comment]
leapp-ci build

**Pull request merged**